packages/perps-exes/assets/config-chain.toml

```diff
@@ -251,3 +251,15 @@ rpc-nodes = ["https://stagenet-thornode.ninerealms.com"]
 gas-decimals = 8
 spot-price = {}
 explorer = "https://stagenet.thorchain.net"
+
+[rujira-mainnet]
+explorer = "https://thorchain.net"
+rpc-nodes = ["https://thornode-mainnet-rpc.bryanlabs.net"]
+gas-decimals = 8
+
+[rujira-mainnet.spot-price.thorchain]
+contract = "thorxxxx"
+
+[rujira-mainnet.assets.USDT]
```

Comment:
We need USDC defined. Please confirm with Mark what the correct denom is for USDC on mainnet.